src/reducers.js

```diff
@@ -22,6 +23,7 @@ export default function createReducer() {
     board: boardReducer,
     communicator: communicatorReducer,
     scanner: scannerProviderReducer,
-    notification: notificationsReducer
+    notification: notificationsReducer,
+    track: AddRecord
```

Comment:
`track` name is too generic, `voiceRecorder` shows better intent.
`addRecord` should be renamed to `voiceRecorderReducer`